R/scale-type.R

```diff
@@ -25,7 +24,10 @@ find_scale <- function(aes, x, env = parent.frame()) {
 # Look for object first in parent environment and if not found, then in
 # ggplot2 namespace environment.  This makes it possible to override default
 # scales by setting them in the parent environment.
-find_global <- function(name, env, mode = "any") {
+find_global <- function(name, env, mode = "any", theme_ = theme()) {
```

Comment:
We don't normally use argument names ending in `_`, as in `theme_`? Does `theme = theme()` not work here?

Reply:
I got `promise already under evaluation`, otherwise I'd have gone with that. I can do `theme=ggplot2::theme()` instead. See reprex:

``` r
x <- "Hello world"
f <- function(get = get("x")) {
  print(get)
}
f()
#> Error in print(get): promise already under evaluation: recursive default argument reference or earlier problems?
```

Reply:
As I commented elsewhere, just write `theme = NULL`.